scripts/realpath

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash -e
```

Comment:
Can you please prefix the filename with an underscore like `_keyid.js`? Ideally it'd be clear from the filenames which ones are proper commands and which ones are helpers.

Reply:
so you mean rename "realpath" to "_realpath" ? @dfunckt

Reply:
@richbayliss yes, “_realpath.sh” even better. I see you renamed it to realpath-helper.sh already but I don’t see why we need to introduce another naming convention. If the underscore annoys you, you could maybe move these two and “ssl-common.sh” into a new “helpers|common|utils” subfolder? Whatever you think is best.

Reply:
I have resolved to the _helper @dfunckt